distro/run/core/src/main/java/org/camunda/bpm/run/CamundaBpmRunDeploymentConfiguration.java

```diff
@@ -25,23 +31,16 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
-import org.apache.commons.lang3.StringUtils;
-import org.camunda.bpm.spring.boot.starter.configuration.impl.DefaultDeploymentConfiguration;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.core.env.Environment;
-import org.springframework.core.io.FileSystemResource;
-import org.springframework.core.io.Resource;
-
 public class CamundaBpmRunDeploymentConfiguration extends DefaultDeploymentConfiguration {
 
-  public static final String CAMUNDA_DEPLOYMENT_DIR_PROPERTY = "camunda.deploymentDir";
+  private final String deploymentDir;
 
-  @Autowired
-  private Environment env;
+  public CamundaBpmRunDeploymentConfiguration(String deploymentDir) {
+    this.deploymentDir = deploymentDir;
+  }
 
   @Override
   public Set<Resource> getDeploymentResources() {
-    String deploymentDir = env.getProperty(CAMUNDA_DEPLOYMENT_DIR_PROPERTY);
```

Comment:
**Explanation**: Externalised the Injection of the property `camunda.deploymentDir` to its respective configuration class.
**Reason**: 
- Less complexity; no need for programmatic handling
- Consistency with the injection of other properties